<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDessert</class>
 <widget class="QDialog" name="DialogDessert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(22, 22, 22)
}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Назва десерту</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	color: #FFF;
	border: 1px solid rbg(255, 255, 255);	
	border-radius: 20%;	
	background-color: rgb(27, 27, 27);
}</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Кількість калорій</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Вміст цукру</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	color: #FFF;
	border: 1px solid rbg(255, 255, 255);	
	border-radius: 20%;	
	background-color: rgb(27, 27, 27);
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Інгредієнти</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	color: #FFF;
	border: 1px solid rbg(255, 255, 255);	
	border-radius: 20%;	
	background-color: rgb(27, 27, 27);
}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	color: #FFF;
	border: 1px solid rbg(255, 255, 255);	
	border-radius: 20%;	
	background-color: rgb(27, 27, 27);
}</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid rbg(100, 100, 100);	
	border-radius: 20%;	
	color: #FFF;
	background-color: rgb(27, 27, 27);
}

QPushButton:hover {
	
}</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid rbg(100, 100, 100);	
	border-radius: 20%;	
	color: #FFF;
	background-color: rgb(27, 27, 27);
}

QPushButton:hover {
	
}</string>
     </property>
     <property name="text">
      <string>Створити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
